htdocs/core/modules/modReceiptPrinter.class.php

```diff
@@ -141,7 +141,7 @@ public function init($options = '')
 		$sql = array(
 			"CREATE TABLE IF NOT EXISTS ".MAIN_DB_PREFIX."printer_receipt (rowid integer AUTO_INCREMENT PRIMARY KEY, name varchar(128), fk_type integer, fk_profile integer, parameter varchar(128), entity integer) ENGINE=innodb;",
 			"CREATE TABLE IF NOT EXISTS ".MAIN_DB_PREFIX."printer_receipt_template (rowid integer AUTO_INCREMENT PRIMARY KEY, name varchar(128), template text, entity integer) ENGINE=innodb;",
-			"DELETE FROM ".MAIN_DB_PREFIX."printer_receipt_template WHERE name = '".$this->db->escape($langs->trans('Example'))."';",
+			"TRUNCATE TABLE ".MAIN_DB_PREFIX."printer_receipt_template;",
```

Comment:
TRUNCATE is an order that break database integrity (no transaction commit/rollback). Can you replace with DELETE